### PR TITLE
docs(installation.md): remove repetitive instrux

### PR DESCRIPTION
**Pull request by d-sanderson**
Removes last sentence of first bullet point under Notes section (installation.md) which seemed repetitive of the 2nd bullet point

### CODE REVIEW

File: docs/usage/installation.md [hunk 1]
```diff
@@ -29,9 +29,8 @@ npx semantic-release
 1. If you've globally installed **semantic-release** then we recommend that you set the major **semantic-release** version to install.
    For example, by using `npx semantic-release@18`.
    This way you control which major version of **semantic-release** is used by your build, and thus avoid breaking the build when there's a new major version of **semantic-release**.
-   This also means you, or a bot, must upgrade **semantic-release** when a new major version is released.
 2. Pinning **semantic-release** to an exact version makes your releases even more deterministic.
-   But pinning also means you, or a bot, must update to newer versions of **semantic-release** more often.
+   But pinning also means you, or a bot, must upgrade **semantic-release** when a new major version is released.
```
Comment (babblebey) [inline docs/usage/installation.md:33]:
```suggestion
   But pinning also means you, or a bot, must upgrade **semantic-release** when a new version is released.
```

Shouldn't have to be a "major" version, I think 🤔

**Review by babblebey — COMMENTED** [review]
Thanks @d-sanderson

Apart from the nit, this looks good to me 🤌🏾

File: docs/usage/installation.md (hunk 1 above)
Comment (d-sanderson) [reply docs/usage/installation.md]: @babblebey good catch man! Added in your suggestion :D

**Review by travi — APPROVED** [review]
thanks for the feedback!

**Pull request merged**